omod/src/main/java/org/openmrs/web/WebComponentRegistrar.java

```diff
@@ -41,7 +41,7 @@ public void setServletContext(ServletContext servletContext) {
 			
 			filter = servletContext.addFilter("forcePasswordChangeFilter", new ForcePasswordChangeFilter());
 			filter.setInitParameter("changePasswordForm", "/admin/users/changePassword.form");
-			filter.setInitParameter("excludeURL", "changePasswordForm,logout,.js,.css,.gif,.jpg,.jpeg,.png");
+			filter.setInitParameter("excludeURL", "csrfguard,logout,.js,.css,.gif,.jpg,.jpeg,.png");
```

Comment:
Yes @dkayiwa. `changePasswordForm` is redundant and is not used on any URL.

Reply:
Is it related to what you are trying to fix?

Reply:
I do not usually recommend mixing things. Just incase you introduce bugs while doing something else. For general improvements, i would rather use a separate pull request.

Reply:
Leaving `changePasswordForm` on the `excludeURL` parameter is as good as changing it to `xxxxxx_foo` as far I've traced the implications to the change I've added. While it might not break anything, it can be one of those potential security loopholes...

Reply:
Except if this `changePasswordForm` was meant to look like `changePassword.form` then I'd be break something. Since that never even became realised all the while `changePasswordForm` is still in the code, it's as good as removing it altogether?

Reply:
The fact that you are asking all these questions means that what you may see as a one line change may turn out not to be what you thought. Fix the original problem. And if you want to investigate this too, use a different pr. :)

Reply:
True dat @dkayiwa.... Fixed... :-)